pages/plans/[id].js

```diff
@@ -26,6 +27,8 @@ const PlanSummary = ({ plan }) => {
     setEditGoal(false);
   };
 
+  const actions = _plan.goal ? _plan.goal.actions : null;
```

Comment:
this can be moved inside the {} where the list is rendered

Reply:
👌